docs/app/demo/button/button-demo.component.ts

```diff
@@ -3,10 +3,7 @@ import { Component } from '@angular/core';
 @Component({
     selector: 'hc-button-demo',
     templateUrl: './button-demo.component.html',
-    styleUrls: [
-        './button-demo.component.scss',
-        '../shared-demo-styles.scss'
```

Comment:
I moved the shared styles into the `styles.scss`  doing this was a little redundant